.github/workflows/wheels.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - "*"
```

Comment:
Can we have a sort of regexp here to ensure we only match tags that look like a version number?

Reply:
Assuming semantic versioning, so something like `[0-9]+.[0-9]+.[0-9]+` ?

Reply:
We don't need the "v" prefix